src/mgmt/storage_query.pb-c.h

```diff
@@ -267,10 +315,14 @@ struct  _Mgmt__SmdQueryReq
    * response should only include information about this VOS target
    */
   char *target;
+  /*
+   * UUID of new device to replace storage with
+   */
+  char *replaceuuid;
```

Comment:
Why 'replace' and 'replaceuuid' are introduced in query reqeust?

Reply:
the replace flag is not used currently, I will remove. The replace UUID is used for the new UUID, and there is already an UUID param that is used for the old UUID.  Both need to be passed

Reply:
I just don't understand why this SmdQueryReq is relative to device replace command.

Reply:
smdquery request is actually used for the smd device set faulty command, and now I added the device replace command to it as well for reuse purposes